src/signals/incident-management/containers/Dashboard/Dashboard.tsx

```diff
@@ -1,23 +1,64 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { useState } from 'react'
+import { useEffect, useMemo, useState } from 'react'
 
 import { Row } from '@amsterdam/asc-ui'
+import { capitalize } from 'lodash'
+import { Link, Route, useLocation, useHistory } from 'react-router-dom'
 
 import { AreaChart, BarChart, Filter } from './components'
-import { StyledRow } from './styled'
+import { StyledRow, StyledTabContainer } from './styled'
+import { Tab } from '../../../../components/Tab'
+import { DASHBOARD_URL } from '../../routes'
 
 const Dashboard = () => {
   const [queryString, setQueryString] = useState<string>('')
 
+  const location = useLocation()
+
+  const history = useHistory()
+
+  const pages = useMemo(
+    () => ({
+      nu: `${DASHBOARD_URL}/nu`,
+      signalering: `${DASHBOARD_URL}/signalering`,
+      vergelijk: `${DASHBOARD_URL}/vergelijk`,
+    }),
+    []
+  )
+
+  useEffect(() => {
+    if (location.pathname === DASHBOARD_URL) {
+      history.push(pages.nu)
+    }
+  }, [history, location.pathname, pages])
+
   return (
     <>
       <StyledRow data-testid="menu">
         <Filter callback={setQueryString} />
       </StyledRow>
-      <Row data-testid="menu">
-        <BarChart queryString={queryString} />
-        <AreaChart />
+      <Row>
```

Comment:
Has conflicts with main. It should use the new "styledRow" which has some more padding on large screens.